### DISCUSSION TITLE
Custom text highlight

### DISCUSSION THREAD

**Discussion by davidelarosakp** [Eugeny/tabby#3719] [General]
Hello,
i'm new in Terminus and i'm still trying to learn all its features.
Something i appreciated so much is the possibility to customize the text colors but, i'd even like to highlight some custom words that could help while reading text like logs or similar.
For example:
i'd like to highlight:
in red all the "ERROR" or "error"
in yellow all the "WARNINGS" or "warnings"
and in green all the "OK", "PASSED", "DONE" and so on.

Is it already possible?
If it's already possible someone could help me to do this?

Thank you so much!

**Comment by Eugeny** [comment]
See #632
